usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -194,8 +203,8 @@ def f(provider):
 @pytest.fixture()
 def get_smt_servers(get_release_value, host):
     def f(provider, region):
-        cpe_name = get_release_value('CPE_NAME')
-        if 'sap' in cpe_name:
+        product_name = get_baseproduct()
```

Comment:
Would be better to move the is_sles_sap determination outside of this fixture and inject the fixture instead.

```
def get_smt_servers(get_release_value, is_sles_sap, host):
    def f(provider, region):
        if is_sles_sap():
            smt_type = 'smt-sap'
        ...
```

Reply:
That way if the route to determining if is SAP image changes it only changes in one location.

Reply:
is_sles_sap would have to move to base conftest so pytest could properly discover and inject it during initialization of module.

Reply:
The other option is to run is_sles_sap from the test itself and only inject a boolean:

```
def get_smt_servers(get_release_value, is_sap, host):
    def f(provider, region):
        if is_sap:
            smt_type = 'smt-sap'
        ...
```

But this can cause problems with fixtures if the arg name overlaps with a fixture name.